packages/fiori/test/pages/NavigationLayout.html

```diff
@@ -0,0 +1,180 @@
+				</p>
+			</div>
+			<div id="item6" class="contentItem">
+				<h2>Item 6</h2>
+				<p>
+					Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+				</p>
+			</div>
+			<div id="item7" class="contentItem">
+				<h2>Item 7</h2>
+				<p>
+					Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+				</p>
+			</div>
+			<div id="subitem1" class="contentItem">
+				<h2>Sub Item 1</h2>
+				<p>
+					Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+				</p>
+			</div>
+			<div id="subitem2" class="contentItem">
+				<h2>Sub Item 2</h2>
+				<p>
+					Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+				</p>
+			</div>
+		</div>
+	</ui5-navigation-layout>
+
+	<script>
+		document.querySelector("#startButton").addEventListener("click", function (event) {
+			nl1.sideCollapsed = !nl1.sideCollapsed;
+		});
+
+		document.querySelector("#sn1").addEventListener("selection-change", function (event) {
+			if (event.detail.item.getAttribute("target")) {
+				return;
+			}
+
+			const contentItems = document.querySelectorAll(".contentItem");
+			contentItems.forEach(item => {
+				item.classList.remove("contentItemVisible");
+			});
+			document.getElementById(event.detail.item.getAttribute("href").replace("#", "")).classList.add("contentItemVisible");
```

Comment:
if you choose "Sub Item 3", "Sub Item 4" or others below them - this line fails with error in console "NavigationLayout.html:186 Uncaught TypeError: Cannot read properties of null (reading 'classList')"